src/freenas/usr/local/bin/ixdiagnose

```diff
@@ -138,29 +145,41 @@ else
 	$mydir/freenas-debug -A
 fi
 
+if is_linux; then
+	h_args="h"
+else
+	h_args="H"
+fi
```

Comment:
Maybe we should be using `h` on both if we're going to start doing it on Linux.

Reply:
@freqlabs there shouldn't be any issues ideally. I'm just trying to avoid touching freebsd's original behaviour as much as I can. Do you see any specific reason for us using that ?

Reply:
Copying the targets of symlinked files instead of just symlinks may be more useful for debugging, and more shared code is a bonus.